TheBosZ/react-json-view
# Fix: make device config card easier to use

**Problem**
Editing fields in the Device Config card is confusing due to multiple checkmarks for changes.

**Solution**
Make it less confusing by moving the cast to string button below and add label.

Testing
1. Use React-Json-view in a project.  
2. Notice there will be two checkmarks when using an identifiable variable type such as a number or boolean.
3. Switch to this branch and notice it is easier to understand.
